azurebatchdockerhelp.md

```diff
@@ -0,0 +1,56 @@
+## Azure Batch with docker containers
+Azure Batch has the functionality to run jobs and tasks within a docker container. Using docker containers to execute REEF jobs has the benefit of setting up the dependencies in light-weight docker containers instead of the vm node. This section describes how you can configure the pool and REEF to execute REEF jobs inside docker containers.
+
+####	1. Create Dockerfile for your OS with REEF and other dependencies 
+##### Option: Windows Docker image
```

Comment:
Add some text explaining why you have to create your own docker container or why they can't use an off the shelf container.